packages/docs/users/login-and-logout.yaml

```diff
@@ -41,13 +41,13 @@ _ref:
             The `Login` action can be used to update the `user` object, for example to update the user profile after it has been edited, or to make sure the user token is still valid before editing or creating a record in a database.
 
             #### Examples
-
-            ###### A login page that redirects users in the onEnter event:
+            # TODO: is onMount a good idea here
```

Comment:
I don't see why not. You want to execute every time the user enters the page. We should change the `Context` to a `Box` though